### DISCUSSION TITLE
[灌水]当你没抽UP池be like

### DISCUSSION THREAD

**Discussion by EhuanyingK** [share121/inter-knot#3407] [其他]
氪了42
![1](https://github.com/user-attachments/assets/ac4ca126-2100-4611-aca2-c55cdfeb12bd)
![2](https://github.com/user-attachments/assets/d5d48744-9836-4556-a163-9f0587f2d6e5)

**Comment by FFlaming0** [comment]
再多母带也顶不住歪一次<img src="https://bbs-static.miyoushe.com/static/2024/03/21/eb81560ecf3e308280f8d3599f24b64d_8586799033270022982.png" alt="丽都漫步-艾莲" width="75" >
